deploy/examples/create-external-cluster-resources.py

```diff
@@ -1793,12 +1801,28 @@ def upgrade_users_permissions(self):
             "client.csi-cephfs-provisioner",
             "client.csi-rbd-node",
             "client.csi-rbd-provisioner",
+            "client.healthchecker",
         ]
         if self.run_as_user != "" and self.run_as_user not in users:
             users.append(self.run_as_user)
         for user in users:
             self.upgrade_user_permissions(user)
 
+    def get_rgw_pool_name_during_upgrade(self,user, caps):
```

Comment:
I see a formating issue `, caps`
Can you run the python black formatter to make the formatting consistent

Reply:
okay